src/app/AppComponent.tsx

```diff
@@ -69,7 +70,13 @@ const AppComponent: React.SFC<IProps> = ({
   updateTokens,
   clusterList,
   debugTree,
+  initializeDiscoveryService,
 }) => {
+  // Early prompt for discovery service cert acceptace if it hasn't already been accepted
+  useEffect(() => {
+    initializeDiscoveryService();
+  }, []);
+
```

Comment:
@eriknelson, not sure if it matters, but is there a reason you put this here and not in the `appStarted` saga?

Reply:
Ah I see, it's probably to make sure resources are loaded first

Reply:
Played around with moving this call to the appStarted saga & looks like everything is still working as expected.